### PR TITLE
add new icons

### PR DESCRIPTION
**Pull request by lucasleal-catho**
## Description
Add new icons

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [X] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [X] Unit tests (yarn test:components)
- [X] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review
- [X] TypeScript updated

### A11y review
- [ ] A11y checks
  - [ ] accessible via keyboard?
  - [ ] Identifiable through assistive technology (screen-reader chrome extension)?
  - [ ] Semantically correct html elements (or if necessary identified by WAI-ARIA)?
  - [ ] Is there a deficiency in color contrast?

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

File: components/Icon/Icon.jsx [hunk 1]
```diff
@@ -201,6 +212,9 @@ const Icon = ({ name, skin, size, ...props }) => {
     star: Star,
     stars: Stars,
     swap_vert: SwapVert,
+    thumb_down_outlined: ThumbDownOutlined,
+    thumb_up_outlined: ThumbUpOutlined,
+    timer_outline: TimerOutlined,
```
Comment (m4ur1c10) [inline components/Icon/Icon.jsx:217]: Acho que aqui deveria ser `timer_outlined`

Comment (lucasleal-catho) [reply components/Icon/Icon.jsx]: Corrigido

**Review by santanasara — APPROVED** [review]

**Review by m4ur1c10 — APPROVED** [review]

**Review by alizeleal — APPROVED** [review]
- [x] Code Review
- [x] Layout
- [x] Unit Tests
- [x] Regression Tests

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review
[x] Doc

**Pull request merged**